tests/e2e/nativeCommands/adbBackspace.ts

```diff
@@ -1,6 +1,7 @@
 import execAsync from '../utils/execAsync';
 import * as Logger from '../utils/logger';
 
+// eslint-disable-next-line @typescript-eslint/require-await
 const adbBackspace = async () => {
```

Comment:
Maybe we can make this function to be sync?
But to make it all work, we need to remove async here and for [adbTypeText](https://github.com/Expensify/App/pull/37676/files#diff-1b96ec4f1bf8ffac3715bfa655747784ada998a8c5b185ca5c0e7e64405e6563R5) function.

And then in this part
https://github.com/Expensify/App/blob/982e2eed47d2268dafa8e23ca1c8ea2c4acc7aa8/tests/e2e/server/index.ts#L143-L149
update it to just
```
const status = nativeCommands.executeFromPayload(data?.actionName, data?.payload);
if (status) {
    res.end('ok');
    return;
}
res.statusCode = 500;
res.end('Error executing command');
```

Reply:
Although I agree with the change we should test it carefully in order to avoid regressions

Reply:
Seems to be working locally fine. @hannojg could ypu also take a look on those changes ?

Reply:
lgtm!